assets/js/modules/analytics-4/utils/validation.js

```diff
@@ -93,3 +94,46 @@ export function isValidMeasurementID( measurementID ) {
 		/^G-[a-zA-Z0-9]+$/.test( measurementID )
 	);
 }
+
+/**
+ * Checks whether the given googleTagId appears to be valid.
+ *
+ * @since n.e.x.t
+ *
+ * @param {*} googleTagID Google Tag ID to check.
+ * @return {boolean} TRUE if the googleTagID is valid, otherwise FALSE.
+ */
+export function isValidGoogleTagID( googleTagID ) {
+	return (
+		typeof googleTagID === 'string' &&
+		/^(G|GT|AW)-[a-zA-Z0-9]+$/.test( googleTagID )
+	);
+}
+
+/**
+ * Checks whether the given googleTagAccountID appears to be valid.
+ *
+ * @alias module:tagmanager/utils/validation.isValidAccountID
```

Comment:
This is to show that it is only a function alias. But I will remove it because it is no longer up to date after the change to a common utility function. I had probably missed that when refactoring.

Reply:
Ah, cool, makes sense. Thanks for the explanation (and for removing it after the refactor). All good 👍🏻